### PR TITLE
feat: applying chart 1 namespace at a time

### PR DESCRIPTION
**Pull request by garethjevans**

### CODE REVIEW

File: pkg/helm/helm_template.go [hunk 1]
```diff
@@ -383,9 +384,47 @@ func (h *HelmTemplate) DecryptSecrets(location string) error {
...
+				args = append(args, "--wait")
+			}
+			if !h.KubectlValidate {
+				args = append(args, "--validate=false")
+			}
+			err = h.runKubectl(args...)
+			if err != nil {
+				return err
+			}
+			log.Info("\n")
```
Comment (ccojocar) [inline pkg/helm/helm_template.go]: is this log needed here?

**Comment by jstrachan** [comment]
gonna close this - I've modified this PR a little to get the tests passing: https://github.com/jenkins-x/jx/pull/4209

**Pull request closed without merging**